src/main/java/assignment/Input.java

```diff
@@ -2,12 +2,12 @@
 
 // Input for the decide function
 public class Input {
-    public final Point[] POINTS;
-    public final Parameters PARAMS;
-    public final Connect[][] LCM;
-    public final boolean[] PUV;
+    final Point[] POINTS;
+    final Parameters PARAMS;
+    final Connect[][] LCM;
+    final boolean[] PUV;
 
-    public static enum Connect {
+    static enum Connect {
```

Comment:
Connect should be public, otherwise it is impossible to construct Input outside of our package.